presto-native-execution/presto_cpp/main/tests/PrestoExchangeSourceTest.cpp

```diff
@@ -382,6 +382,7 @@ class PrestoExchangeSourceTestSuite : public ::testing::TestWithParam<bool> {
     options.capacity = 1L << 30;
     allocator_ = std::make_unique<memory::MmapAllocator>(options);
     memory::MemoryAllocator::setDefaultInstance(allocator_.get());
+    executor_ = std::make_shared<folly::IOThreadPoolExecutor>(10);
```

Comment:
Consider to use parameterized test to test two settings: 1 and 10 threads. Thanks!